fvm/executionParameters.go

```diff
@@ -24,6 +24,26 @@ type MeterParamOverrides struct {
 	MemoryLimit        *uint64                      // nil indicates no override
 }
 
+func getBasicMeterParameters(
+	ctx Context,
+	proc Procedure,
+) meter.MeterParameters {
+	params := meter.DefaultParameters().
+		WithComputationLimit(uint(proc.ComputationLimit(ctx))).
+		WithMemoryLimit(proc.MemoryLimit(ctx)).
+		WithEventEmitByteLimit(ctx.EventCollectionByteSizeLimit).
+		WithStorageInteractionLimit(ctx.MaxStateInteractionSize)
+
+	// NOTE: The memory limit (and interaction limit) may be overridden by the
+	// environment.  We need to ignore the override in that case.
```

Comment:
why do we need to ignore the override?

If we need to ignore the override, why do we override it in the first place?

Reply:
This is prep work for what i've described above.  you can think of the overrides in terms of priorities:

1. ShouldDisableMemoryAndInteractionLimits has the highest priority
2. Environment meter overrides have the medium priority
3. Basic meter parameters has the lowest priority